somecode.cpp

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+#include <string.h>
+
+int main()
+{
+   char src[40];
+   char dest[100];
+   char dest2[100];
+
+   printf("Enter your name");
+   gets(src); 
+
+   // Copy name from one place to another
+   strcpy(dest, src); 
+ 
+   int a = 10;
+   int b = 20;
+   int c = a+b;
+   printf("Result is %d",&c);
```

Comment:
[Banned C function detected](https://github.com/microsoft/devskim/guidance/DS154189.md)
These functions are historically error-prone and have been associated with a significant number of vulnerabilities. Most of these functions have safer alternatives, such as replacing 'strcpy' with 'strlcpy' or 'strcpy_s'.

*Replacement:*